packages/ella/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@groww-tech/ella",
-  "version": "0.2.4",
+  "version": "0.2.5",
```

Comment:
Have increased this version as there is a dependency on ella

Reply:
If dependency on Ella then add Ella as dependency, why increasing the version?